<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QMainWindow" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>793</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1212</width>
    <height>793</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1212</width>
    <height>793</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="add_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>添加学生</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_info">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>删除学生</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_data">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>保存数据</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>1211</width>
      <height>631</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1211</width>
      <height>631</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1211</width>
      <height>631</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;隶书&quot;;</string>
    </property>
    <property name="text">
     <string>教师管理</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>搜索</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="currentText">
     <string>全校</string>
    </property>
    <item>
     <property name="text">
      <string>全校</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>软件1401</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>计算机1401</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>计算机1402</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="currentText">
     <string>全科</string>
    </property>
    <item>
     <property name="text">
      <string>全科</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>语文</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>数学</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>英语</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>72</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>专业班级</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>75</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>学科</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="currentText">
     <string>默认</string>
    </property>
    <item>
     <property name="text">
      <string>默认</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按学号顺序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按班级顺序</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按语文成绩</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按数学成绩</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>按英语成绩</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>70</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;楷体&quot;;</string>
    </property>
    <property name="text">
     <string>排序</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="tableView">
   <property name="text">
    <string>tableView</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
